packages/search-ui-site-search-connector/bin/build-js

```diff
@@ -1,5 +1,7 @@
-# Builds Common JS JavaScript
-babel src --out-dir lib --ignore '**/*.test.js'
+# clear directory
+rm -rf ./lib
```

Comment:
you can take out the rm -rf as its been removed by rimraf in the NPM scripts

Reply:
Nice catch! https://github.com/elastic/search-ui/pull/637/commits/386a32437aa3eceb3047ea4d93e9623030ccf347